batch_scripts/run_inference_standard_slurm.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-#SBATCH -N 60
+#SBATCH -N 116
```

Comment:
116? That's a really weird number. Why?

Reply:
write a comment that says it allocates 16 gpu nodes and 100 cpu nodes